### ISSUE TITLE
Be consistent with categorisation of pearl models

### ISSUE THREAD

**Issue by smk78** [SasView/sasmodels#426]
The pearl_necklace model is categorised as a cylinder function, but the linear_pearls model is categorised as a sphere function. This seems arbitrary. We should be consistent.

**Comment by butlerpd** [comment]
categories are defined as tags really so that any model can belong to several categories.  This was envisaged for grouping in different ways for projects for example but could be applied here in that both models could be categorized as both sphere and cyclinder?

**Comment by smk78** [comment]
I understand what you are saying @butler, but I still think it is inconsistent that a linear string of spheres is categorised as a 'sphere' model but a non-linear string of spheres is a 'cylinder'! If a user were searching for one or other models they could easily miss it as things stand.

**Comment by butlerpd** [comment]
Agreed things should be consistant. That is clear. I meant to be proposing a solution to the fact that clearly different people think they should be categorized differently (I assume that is how the discrepancy came about?). That said I am not sure if a model can be defined straight up as belonging to two "built-in" categories or not, in which case, either we should change that or my proposal is not a solution 😃
